go/account_server/main.go

```diff
@@ -97,6 +100,20 @@ func main() {
 		serverOpts = append(serverOpts, grpc.StatsHandler(&ocgrpc.ServerHandler{}))
 	}
 
+	// Start admin server
+	adminListener, err := net.Listen("tcp", "localhost:"+args.adminPort)
+	if err != nil {
+		log.Fatalf("failed to listen: %v", err)
+	}
+	adminServer := grpc.NewServer()
+	cleanup, err := admin.Register(adminServer)
```

Comment:
Do we need reflection and health on the admin server as well?
I see Java (and CPP?) has it, but not Go.

Reply:
According to Doug, reflection and health are per-server services. We don't want to force users to register reflection and health as part of admin services, because they could query the wrong instance of the reflection service or health service (e.g., the business server's overall state is NOT_SERVING, and admin server's overall state is SERVING).

Removed the reflection and health services from C++/Java admin servers.